futbin-club-worth.user.js

```diff
@@ -13,51 +13,61 @@
 // @supportURL  https://github.com/Mardaneus86/futwebapp-tampermonkey/issues
 // ==/UserScript==
 // ==OpenUserJS==
-// @author Mardaneus86
+// @author Mardaneus86, Kyderman
```

Comment:
This is used by OpenUserJS for collaboration on their platform, no need to add your name here.